app/views/userTournament/list.scala

```diff
@@ -18,17 +18,17 @@ object list:
       pager: Paginator[lila.tournament.LeaderboardApi.TourEntry],
       count: String
   )(using Lang) =
-    if pager.nbResults == 0 then div(cls := "box-pad")(u.username, " hasn't played in any tournament yet!")
+    if pager.nbResults == 0 then div(cls := "box-pad")(trans.arena.xHasNotPlayedInAnyTourYet(u.username))
```

Comment:
this is also broken. Please try your change when publishing a pull request, or set it to "DRAFT" until it's tested.